app/views/layouts/_footer.html.erb

```diff
@@ -68,8 +68,8 @@
         }, Rails.configuration.branding[:organisation][:url])%></li>
       <li>•</li>
       <li><a href="<%= contact_us %>"><%= _('Contact us') %></a></li>
-      <li><a href="<%= terms_of_use_path %>"><%= _('Terms of use') %></a></li>
-      <li><a href="<%= privacy_path %>"><%= _('Privacy statement') %></a></li>
+      <li><a href="/terms"><%= _('Terms of use') %></a></li>
+      <li><a href="/privacy"><%= _('Privacy statement') %></a></li>
```

Comment:
We should not be hardcoding the URLs whenever possible. Is there a reason for this change?

Reply:
yeah, it used to link to an external portagenetwork site. now it needs to link to static page